docs-gen/layouts/shortcodes/piechart.html

```diff
@@ -110,59 +109,60 @@
-radioForm.style.display = "inline-block"; // Ensure form is inline-block for center alignment
-
-// Add a label before the radio buttons
-const formLabel = document.createElement("span");
-formLabel.textContent = "Release: ";
-formLabel.style.fontWeight = "500";
-formLabel.style.marginTop = "20px";
-formLabel.style.marginRight = "10px";
-radioForm.appendChild(formLabel);
-
-datasets.forEach((value, key) => {
-  const label = document.createElement("label");
-  label.style.marginLeft = "15px"; // Add space between options
-  label.style.fontWeight = "500"; // Semibold text style
-  label.style.display = "inline-block"; // Ensure radio buttons are aligned horizontally
-
-  const input = document.createElement("input");
-  input.type = "radio";
-  input.name = "dataset";
-  input.value = value;
-
-  if (value === "V5") input.checked = true; // Default selection
-  input.addEventListener("change", () => {
-    console.log(`Dataset changed to: ${value}`);
-    updateChart(value);  // Trigger chart change based on the selected value
-  });
-
-  label.appendChild(input);
-  label.appendChild(document.createTextNode(key));
-
-  radioForm.appendChild(label);
-});
-
-selectorContainer.appendChild(radioForm);
-console.log("Dataset selector created successfully");
+    const radioForm = document.createElement("form");
+    radioForm.style.display = "inline-block"; // Ensure form is inline-block for center alignment
+
+    // Add a label before the radio buttons
+    const formLabel = document.createElement("span");
+    formLabel.textContent = "Release: ";
+    formLabel.style.fontWeight = "500";
+    formLabel.style.marginTop = "20px";
+    formLabel.style.marginRight = "10px";
+    radioForm.appendChild(formLabel);
+
+    datasetKeys.forEach(key => {
+      const label = document.createElement("label");
+      label.style.marginLeft = "15px"; // Add space between options
+      label.style.fontWeight = "500"; // Semibold text style
+      label.style.display = "inline-block"; // Ensure radio buttons are aligned horizontally
+
+      const input = document.createElement("input");
+      input.type = "radio";
+      input.name = "dataset";
+      input.value = key;
+
+      if (key === latestVersion) input.checked = true; // Default selection
+      input.addEventListener("change", () => {
+        console.log(`Dataset changed to: ${key}`);
+        updateChart(key);  // Trigger chart change based on the selected value
+      });
+
+      label.appendChild(input);
+      label.appendChild(document.createTextNode(key));
+
+      radioForm.appendChild(label);
+    });
+
+    selectorContainer.appendChild(radioForm);
+    console.log("Dataset selector created successfully");
   }).catch(error => {
     console.error("Error loading CSV data:", error);
     document.getElementById("donut").innerText = "Error loading data. Please check the console for details.";
   });
 
   console.log("Script execution complete");
-</script>
+</script>
```

Comment:
I think this the reason that pre-commit complains, that it as of today expects a line break at end of file